interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -296,9 +296,15 @@ public void setUp() {
 
   /** Clean up. */
   @After
-  public void tearDown() throws Exception {
+  public void tearDown() {
     if (channel != null) {
-      channel.shutdownNow().awaitTermination(1, TimeUnit.SECONDS);
+      channel.shutdownNow();
+      try {
+        channel.awaitTermination(1, TimeUnit.SECONDS);
+      } catch (InterruptedException unused) {
+        // Best effort. If there is an interruption, we want to continue cleaning up, but quickly
```

Comment:
this should be logged.

Reply:
Done, at FINE level. Since we do `shutdownNow()` I had considered just removing the awaitTermination completely.